src/gui/src/main.cpp

```diff
@@ -63,14 +62,22 @@ void copy_qsettings(const QSettings &src, QSettings &dst)
 
 int main(int argc, char* argv[])
 {
-#ifdef WINAPI_XWINDOWS
-    // QApplication's constructor will call a fscking abort() if
-    // DISPLAY is bad. Let's check it first and handle it gracefully
-    if (!display_is_valid()) {
-        fprintf(stderr, "The InputLeap GUI requires the DISPLAY environment variable to be set. Quitting...\n");
-        return 1;
+#if (defined(WINAPI_XWINDOWS) || \
+    (defined(HAVE_LIBPORTAL_INPUTCAPTURE) || \
+    defined(HAVE_LIBPORTAL_SESSION_CONNECT_TO_EIS)))
+    const auto platformType = QGuiApplication::platformName();
+
+    if (platformType == "xcb") {
```

Comment:
Please add an else here. There are more platform plugins that Qt can load on linux for these other ones you may want to log or do maybe `QTextstream(stdout) << "The " << platformType << " plugin is not supported";` . Before calling exit

Reply:
It can load other plugins, but do we need to handle those? Before this PR, all that was handled was Wayland, and X11. I don't think there's any need to handle things specifically, tbh. We could log QPA, but I'm not convinced we need to. Input Leap is noisy in logs enough as it is.

Reply:
No we don't need to handle them but those plugins are not going to talk to a wayland or X11 server and i think in this case its best for us to just report this platform plugin was loaded and exit.

Reply:
It's merged now, but I didn't implement reporting the platform - logging is noisy enough as it is.